codec-mqtt/src/main/java/io/netty/handler/codec/mqtt/MqttMessage.java

```diff
@@ -61,7 +54,7 @@ public Object variableHeader() {
     }
 
     public Object payload() {
```

Comment:
@Scottmitch for whatever reason I thought we used `@UnstableApi` here... damn :(